cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs

```diff
@@ -222,6 +223,72 @@ mod bridge_hub_westend_tests {
 		)
 	}
 
+	#[test]
+	fn change_ethereum_nonces_by_governance_works() {
+		let channel_id_one: ChannelId = [1; 32].into();
+		let channel_id_two: ChannelId = [2; 32].into();
+		let nonce = 42;
+
+		// Reset a single inbound channel
+		bridge_hub_test_utils::test_cases::set_storage_keys_by_governance_works::<Runtime>(
+			collator_session_keys(),
+			bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			Box::new(|call| RuntimeCall::System(call).encode()),
+			vec![
+				(snowbridge_pallet_outbound_queue::Nonce::<Runtime>::hashed_key_for::<ChannelId>(
+					channel_id_one,
+				)
+				.to_vec(), 0u64.encode()),
+				(snowbridge_pallet_inbound_queue::Nonce::<Runtime>::hashed_key_for::<ChannelId>(
+					channel_id_one,
```

Comment:
```suggestion
					channel_id_two,
```

@claravanstaden typo or intention?

Reply:
I think this is intentional because if I change it, the test fails. @alistair-singh can provide more context here and perhaps add a comment why.

Reply:
This test assets that only `channel_one` nonces are reset and that `channel_two` nonces are left alone.